<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>tigervnc.git - tigervnc</title>
<meta name='generator' content='cgit v0.11.2'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/tigervnc.git/atom/tigervnc-xorg118-QueueKeyboardEvents.patch?h=master' type='application/atom+xml'/>
<link rel='vcs-git' href='git://pkgs.fedoraproject.org/tigervnc.git' title='tigervnc.git Git repository'/>
<link rel='vcs-git' href='ssh://pkgs.fedoraproject.org/tigervnc.git' title='tigervnc.git Git repository'/>
<link rel='vcs-git' href='http://pkgs.fedoraproject.org/git/tigervnc.git' title='tigervnc.git Git repository'/>
</head>
<body>
<script src="https://apps.fedoraproject.org/fedmenu/js/jquery-1.11.2.min.js"></script>
<script src="https://apps.fedoraproject.org/fedmenu/js/fedmenu.js"></script>

<script>
    var base = 'https://apps.fedoraproject.org/';
    var tokens = window.location.href.split('/');
  var repo = null;
  if (tokens.length > 4 && tokens[4] != '') {
    repo = tokens[4].split('.')[0];
  }
  fedmenu({
      'url': base + 'js/data.js',
      'mimeType': 'application/javascript',
      'position': 'bottom-right',
      'package': repo,
  });
</script>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='tigervnc.git' href='/cgit/tigervnc.git/'>tigervnc.git</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='el5'>el5</option>
<option value='f11'>f11</option>
<option value='f12'>f12</option>
<option value='f13'>f13</option>
<option value='f14'>f14</option>
<option value='f15'>f15</option>
<option value='f16'>f16</option>
<option value='f17'>f17</option>
<option value='f18'>f18</option>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='f22'>f22</option>
<option value='f23'>f23</option>
<option value='master' selected='selected'>master</option>
<option value='private-twaugh-bz1137023-f20'>private-twaugh-bz1137023-f20</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>tigervnc</td><td class='sub right'>Jesse Keating</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/tigervnc.git/'>summary</a><a href='/cgit/tigervnc.git/refs/'>refs</a><a href='/cgit/tigervnc.git/log/tigervnc-xorg118-QueueKeyboardEvents.patch'>log</a><a class='active' href='/cgit/tigervnc.git/tree/tigervnc-xorg118-QueueKeyboardEvents.patch'>tree</a><a href='/cgit/tigervnc.git/commit/tigervnc-xorg118-QueueKeyboardEvents.patch'>commit</a><a href='/cgit/tigervnc.git/diff/tigervnc-xorg118-QueueKeyboardEvents.patch'>diff</a><a href='/cgit/tigervnc.git/stats/tigervnc-xorg118-QueueKeyboardEvents.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/tigervnc.git/log/tigervnc-xorg118-QueueKeyboardEvents.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/tigervnc.git/tree/'>root</a>/<a href='/cgit/tigervnc.git/tree/tigervnc-xorg118-QueueKeyboardEvents.patch'>tigervnc-xorg118-QueueKeyboardEvents.patch</a></div><div class='content'>blob: 4eb738b791cee91551ecf74de4aa35305984e7b8 (<a href='/cgit/tigervnc.git/plain/tigervnc-xorg118-QueueKeyboardEvents.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
</pre></td>
<td class='lines'><pre><code>diff --git a/unix/xserver/hw/vnc/Input.c b/unix/xserver/hw/vnc/Input.c
--- a/unix/xserver/hw/vnc/Input.c
+++ b/unix/xserver/hw/vnc/Input.c
@@ -301,7 +301,7 @@ static inline void pressKey(DeviceIntPtr dev, int kc, Bool down, const char *msg
 	n = GetKeyboardEvents(eventq, dev, action, kc);
 	enqueueEvents(dev, n);
 #else
-	QueueKeyboardEvents(dev, action, kc, NULL);
+	QueueKeyboardEvents(dev, action, kc);
 #endif
 }
 
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated by <a href='http://git.zx2c4.com/cgit/about/'>cgit v0.11.2</a> at 2015-11-12 22:24:09 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
